apple/RNGestureHandlerButton.mm

```diff
@@ -135,13 +135,16 @@ - (NSString *)accessibilityLabel
 // Vendored from RCTView.m to infer accessibility label from children
 static NSString *RNGHRecursiveAccessibilityLabel(UIView *view)
 {
-  NSMutableString *str = [NSMutableString stringWithString:@""];
+  NSMutableString *str = nil;
```

Comment:
The `return` expression should be updated as well - `str` may not be initialized by then.

Reply:
Good catch, done in [026f2e7](https://github.com/software-mansion/react-native-gesture-handler/pull/3499/commits/026f2e7e426faad4d16165962e25faa8d272703b)